bin/ci-wait-steady-state

```diff
@@ -1,11 +1,23 @@
-# will return also on failing conditions such as dead processes
-timeout 60 ./bin/wait-for-empty-queues &
-waitingPid=$!
-wait "$waitingPid"
-waitingReturnCode=$?
-# clean up PHP processes
-kill "$workerPid" "$queueWatchPid";
-if [ ! "$waitingReturnCode" -eq "0" ]; then
-    echo -e "\e[31mThe script './wait-for-empty-queues' timed out\e[39m"
-    exit "$waitingReturnCode"
-fi
+#!/bin/bash
+
+echo -n "Waiting for queues to empty..."
+for i in $(seq 60); do
+  echo -n "."
+
+  sqs_queue=$(bin/console queue:count || true)
+  gearman_queue=$(gearadmin --show-jobs | wc -l || true)
+  if [ "$sqs_queue" -eq "0" ] && [ "$gearman_queue" -eq "1" ]; then
```

Comment:
magic literal of `1`

why are we waiting for a value of 1 and not 0 (like `$sqs_queue`) when we're emptying queues? If it's deliberate please add a comment

Reply:
Good spot, it is intentional so have added a comment to capture why.